source/guides/administrator.rst

```diff
@@ -58,7 +58,8 @@ Administration
    /administration/compliance*
    /administration/branding*
    /administration/backup*
-   /administration/upgrade.md
+   /administration/upgrade.rst
+   /administration/legacy-upgrade.md
```

Comment:
I don't think including the legacy guide in TOC is necessary. These are really old builds, the vast majority of deployments are on 3.0 and later